src/writer.cc

```diff
@@ -70,60 +72,60 @@ void wabt_init_file_writer_existing(WabtFileWriter* writer, FILE* file) {
   writer->base.move_data = move_data_in_file;
 }
 
-WabtResult wabt_init_file_writer(WabtFileWriter* writer, const char* filename) {
+Result init_file_writer(FileWriter* writer, const char* filename) {
   FILE* file = fopen(filename, "wb");
   if (!file) {
     ERROR("fopen name=\"%s\" failed, errno=%d\n", filename, errno);
-    return WabtResult::Error;
+    return Result::Error;
   }
 
-  wabt_init_file_writer_existing(writer, file);
-  return WabtResult::Ok;
+  init_file_writer_existing(writer, file);
+  return Result::Ok;
 }
 
-void wabt_close_file_writer(WabtFileWriter* writer) {
+void close_file_writer(FileWriter* writer) {
   fclose(writer->file);
 }
 
-void wabt_init_output_buffer(WabtOutputBuffer* buf, size_t initial_capacity) {
+void init_output_buffer(OutputBuffer* buf, size_t initial_capacity) {
   assert(initial_capacity != 0);
-  buf->start = wabt_alloc(initial_capacity);
+  buf->start = alloc(initial_capacity);
```

Comment:
Again, just questioning whether a wabt:: prefix is appropriate so that it doesn't accidentally use "C" alloc  (which could happen by including another standard C header).